### ISSUE TITLE
we should split API and version in profile

### ISSUE THREAD

**Issue by fabrobinet** [KhronosGroup/glTF#325]
Just to make parsing easier.
instead of:
```    
"profile": "WebGL 1.0.2", 
```
have
```
"profile": {
  "API" : "WebGL",
  "version" : "1.0.2"
}
```
a minor change worth doing now IMO.

**Comment by pjcozzi** [comment]
This can be closed when 1.0 is merged into master...or now.

**Comment by pjcozzi** [comment]
This was updated.
